dm/relay/binlog_writer.go

```diff
@@ -55,14 +58,16 @@ func (s *BinlogWriterStatus) String() string {
 }
 
 // NewBinlogWriter creates a BinlogWriter instance.
-func NewBinlogWriter(logger log.Logger) *BinlogWriter {
+func NewBinlogWriter(logger log.Logger, relayDir string) *BinlogWriter {
```

Comment:
Prefer to define a `Relayer` interface and implement the `FsRelayer` for it because we may support `S3Relayer` in the future.

Reply:
maybe in another pr.